cpp/examples/abm.cpp

```diff
@@ -466,8 +466,8 @@ int main()
 
     // The results are saved in a table with 9 rows.
     // The first row is t = time, the others correspond to the number of people with a certain infection state at this time:
-    // S = Susceptible, E = Exposed, C= Carrier, I= Infected, I_s = Infected_Severe,
-    // I_c = Infected_Critical, R_C = Recovered_Carrier, R_I = Recovered_Infected, D = Dead
+    // S = Susceptible, E = Exposed, I_n = InfectedNoSymptoms, I = InfectedSymptoms, I_s = Infected_Severe,
```

Comment:
We should maybe go with
I_NS
I_Sy
I_Sev
I_Crit

Not the best option either by I think better than having no index for Infected Symptoms. I could be understood as the sum of all of these and maybe even should be.

Reply:
I agree. I will adopt this.

Reply:
Changed.